src/custom/components/Header/URLWarning/index.tsx

```diff
@@ -1,9 +1,41 @@
-import { PRODUCTION_URL } from 'constants/index'
 import React from 'react'
-import URLWarningUni from './URLWarningMod'
+
+import { PRODUCTION_URL } from 'constants/index'
+import { AlertTriangle } from 'react-feather'
+import URLWarningUni, { StyledClose } from './URLWarningMod'
+import ReactMarkdown from 'react-markdown'
+import { useAnnouncementVisible, useCloseAnnouncement } from '@src/custom/state/userMod/hooks'
+import { hashCode } from 'utils/misc'
+import useFetchFile from 'hooks/useFetchFile'
 
 export * from './URLWarningMod'
 
+// https://github.com/gnosis/cowswap/blob/announcements/docs/announcements.md
```

Comment:
?

```suggestion
```

Reply:
It's meant to be used for easily finding the file, so u can modify it from Github directly

Reply:
I'll make the comment more explicit.